sentry_sdk/integrations/boto3.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import absolute_import
+
+from sentry_sdk import Hub
+from sentry_sdk.integrations import Integration
+from sentry_sdk.tracing import Span
+
+from sentry_sdk._functools import partial
+from sentry_sdk._types import MYPY
+
+if MYPY:
+    from typing import Any
+    from typing import Dict
+    from typing import Optional
+    from typing import Type
+
+
+class Boto3Integration(Integration):
+    identifier = "boto3"
+
+    @staticmethod
+    def setup_once():
+        # type: () -> None
+        from botocore.client import BaseClient  # type: ignore
+        from botocore.response import StreamingBody  # type: ignore
+
+        if MYPY:
+            from botocore.awsrequest import AWSRequest  # type: ignore
+
+        def sentry_request_created(service_id, request, operation_name, **kwargs):
+            # type: (str, AWSRequest, str, **Any) -> None
+            hub = Hub.current
```

Comment:
Please add a `Hub.current.get_integration` check here like in other integrations to allow the integration to be disabled at a later point in time